packages/components/src/__future__/Tag/Tag/_docs/TagMigration.stories.tsx

```diff
@@ -0,0 +1,321 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { fn } from "@storybook/test"
+import { Avatar } from "~components/Avatar"
+import {
+  CautionIcon,
+  ExclamationIcon,
+  InformationIcon,
+  SuccessIcon,
+  LiveIcon,
+} from "~components/Icon"
+import { Tag, RemovableTag } from "../../"
+import LegacyStyles from "./TagLegacy.module.scss"
```

Comment:
```suggestion
import legacyStyles from "./TagLegacy.module.scss"
```

Shouldn't the styles actually exist in `Tag.module.scss` already though?